doc/living_in_belgium.md

```diff
@@ -0,0 +1,194 @@
+
+## Registration at the commune
+- You probably already have a Visa issued by the Belgian Embassy in your country. This visa allows you to enter Belgium, but it is not the same as a residence permit. You have to register at the local authorities (called "the commune") to receive your residence permit. The duration of the residence permit is typically tied to your UCL work contract (e.g., 2 years).
+
+- On the side note: If you are already residence in any EU countries but you are not EU citizen, you do not need to apply for a visa. When you arrive to Brussels you can directly apply for residence permit. You have 3 months time to apply for this residence permit. (Your residence permit from another EU country is valid for 3 months in Belgium)
+
+- You will need to visit the foreigners office in your local commune within eight days of arriving in Belgium. Note that different areas of Belgium are governed by different communes – there is one for Ottignies-Louvain la Neuve, one for Bruxelles, one for Ixelles, etc. Go to the one that is for your area!
+
+- To register at the commune you will need: Your identification (passport), about 5-8€, some passport size photos of yourself, your work contract, proof of your address (ideally a lease contract), and your legalised marriage certificate if you have one.
+
+- After your initial registration at the commune, they will send a police officer to your residential address to check that you really live there. They usually will not fix a time for this, it’s random. If there are other people living there, explain to them that this is happening and they can try to vouch for you if you are not in the house when the police come. Once you are confirmed to be living there, they will send a letter with the PIN/PUK codes for your card to that address. You go back to the commune with this letter and collect your card. The whole process takes about four weeks at least…
+
+- More general relevant info for Louvain-la-Neuve https://www.uclouvain.be/en-355099.html and for Bruxelles http://be.brussels/living-in-brussels.
+
+
+## Transportation
+
+
+### Commute Brussels-Louvain-la-Neuve
+Try not to advance money when buying a subscription. First ask the UCL for the form to obtain the subscription and THEN buy it. Otherwise you might not get fully reimbursed.
+
+You can ask [Veronique Vanhoegaerden](veronique.vanhoegaerden@uclouvain.be) from UCL. 
+
+
+If you're FNRS, contact [Aurelie Volont](aurelie.volont@uclouvain.be) to ask for the subscription: "Following the agreements with SNCB, it was agreed that the UClouvain could issue the form for obtaining the SNCB subscription. This ensures that the amount does not have to be advanced, since the invoice is sent directly to the FNRS.In this case the subscription is 100% refunded as it is the system of the paying third part of which 80% by the FNRS and 20% by the state. 

```

Comment:
```suggestion
If you're FNRS, contact Aurelie Volont (aurelie.volont@uclouvain.be) to ask for the subscription: "Following the agreements with SNCB, it was agreed that the UClouvain could issue the form for obtaining the SNCB subscription. This ensures that the amount does not have to be advanced, since the invoice is sent directly to the FNRS.In this case the subscription is 100% refunded as it is the system of the paying third part of which 80% by the FNRS and 20% by the state. 

```